templates/repo/diff/box.tmpl

```diff
@@ -1,7 +1,10 @@
 {{if .DiffNotAvailable}}
 	<div>
-		<div class="diff-detail-box diff-box sticky">
-			<div class="ui right">
+		<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
+			<div class="gt-df gt-ac gt-fw">
```

Comment:
Does these `gt-` patches make sense? I guess it doesn't need to fill `gt-` patches to every element if it not used.

And, is the `div` wrapper necessary? `h4` can be there for first element.

Reply:
I was trying to get the same markup as used when there are changes to make sure in the future the layout is nearly the same.

I'll try to follow silverwinds request and try to not duplicate the markup (as it was before this PR, too) and rather use some if statements.

Reply:
Yes, it's best to never duplicate HTML in if/else branches, we should eliminate all such cases.

Reply:
I‘ve removed the duplicate code now